src/snowflake/cli/api/commands/flags.py

```diff
@@ -163,7 +163,7 @@ def callback(value):
     ),
     show_default=False,
     rich_help_panel=_CONNECTION_SECTION,
-    autocompletion=lambda: list(get_all_connections()),
+    shell_complete=lambda _, __, ___: list(get_all_connections()),
```

Comment:
Is this connected to pydantic deprecations? If not, could you update changes description?

Reply:
updated

Reply:
Thanks!